tests/daysOfTheWeek.js

```diff
@@ -2,9 +2,7 @@ const tap = require('tap');
 const target = require('../lib/daysOfTheWeek');
 
 tap.test('getDayName', (getDayNameTest) => {
-  getDayNameTest.throws(() => target.getDayName(), 'throws when using no input');
   getDayNameTest.throws(() => target.getDayName(null), 'throws when using a null input');
-  getDayNameTest.throws(() => target.getDayName(undefined), 'throws when using an undefined input');
   getDayNameTest.throws(() => target.getDayName('string'), 'throws when using a string input');
```

Comment:
Because the behavior of `getDayName` was changed so that it's now okay to call it with no inputs, these two tests are no longer relevant.  Good thing they were removed!